tests/test_scrubber.py

```diff
@@ -169,3 +169,22 @@ def test_scrubbing_doesnt_affect_local_vars(sentry_init, capture_events):
     (frame,) = frames
     assert frame["vars"]["password"] == "[Filtered]"
     assert password == "cat123"
+
+
+def test_recursive_EventScrubber(sentry_init, capture_events):
```

Comment:
Making this more pythonic:
```suggestion
def test_recursive_event_scrubber(sentry_init, capture_events):
```

Also, if you decide to keep the `customScrubber` as a variable (you might not need to, see my other comment on this test), please rename that to `custom_scrubber`.

Reply:
TIL: new word, pythonic